WalletWasabi.Tests/UnitTests/BlockchainAnalysis/PubKeyReuseAnonScoreTests.cs

```diff
@@ -202,9 +231,10 @@ public void OutputSideAddressReuseBySomeoneElse()
 			var analyser = ServiceFactory.CreateBlockchainAnalyzer();
 			var km = ServiceFactory.CreateKeyManager();
 			var equalOutputAmount = Money.Coins(1m);
-			var reusedTxOut = new TxOut(equalOutputAmount, new Key());
+			var destination = new Key();
```

Comment:
```suggestion
			using var destination = new Key();
```

Instead of the dispose at the end.

Reply:
Done